src/Lib.P2P/Multiplex/Header.cs

```diff
@@ -0,0 +1,114 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using MultiFormats;
+
+namespace Lib.P2P.Multiplex
+{
+    /// <summary>
+    ///   The header of a multiplex message.
+    /// </summary>
+    /// <remarks>
+    ///   The header of a multiplex message contains the <see cref="StreamId"/> and
+    ///   <see cref="PacketType"/> encoded as a <see cref="Varint">variable integer</see>.
+    /// </remarks>
+    /// <seealso href="https://github.com/libp2p/mplex"/>
+    public struct Header
+    {
+        /// <summary>
+        ///   The largest possible value of a <see cref="StreamId"/>.
+        /// </summary>
+        /// <value>
+        ///   long.MaxValue >> 3.
+        /// </value>
+        public const long MaxStreamId = long.MaxValue >> 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)